Cython/Utility/ObjectHandling.c

```diff
@@ -2082,8 +2081,7 @@ static PyObject* __Pyx_PyFunction_FastCallNoKw(PyCodeObject *co, PyObject **args
 }
 
 
-#if 1 || PY_VERSION_HEX < 0x030600B1
-static PyObject *__Pyx_PyFunction_FastCallDict(PyObject *func, PyObject **args, int nargs, PyObject *kwargs) {
+static CYTHON_UNUSED PyObject *__Pyx_PyFunction_FastCallDict(PyObject *func, PyObject **args, int nargs, PyObject *kwargs) {
```

Comment:
Can we somehow avoid the `CYTHON_UNUSED` here (if it's not too much hassle)? Better not generate code, than telling _some_ C compilers to silently ignore it.